processing/src/main/java/io/druid/query/search/SearchQueryRunner.java

```diff
@@ -161,7 +161,7 @@ public SearchQueryRunner(Segment segment)
 
             Map<String, DimensionSelector> dimSelectors = Maps.newHashMap();
             for (String dim : dimsToSearch) {
-              dimSelectors.put(dim, cursor.makeDimensionSelector(dim));
+              dimSelectors.put(dim, cursor.makeDimensionSelector(dim, null));
```

Comment:
Can makeDimensionSelector simply take a dim argument and wrap `.makeDimensionSelector(dim, null)` (no extraction function) instead of having to have null all over the place?

Just looking at this function without knowing about the dim extraction function parameter makes the operation of the method less obvious.